tests/foreman/api/test_role.py

```diff
@@ -154,8 +211,23 @@ def test_positive_create_filter_without_override(self):
 
         :CaseImportance: Critical
         """
+        role_name = gen_string('alpha')
+        role1 = entities.Role(
+            name=role_name,
+            organization=[self.org1],
+            location=[self.loc1]
+        ).create()
+        self.assertEqual(role1.name, role_name)
+        dom_perm = (entities.Permission(resource_type='Domain').search())
```

Comment:
Any reason for extra brackets? I think you're safe to remove them

Reply:
Its OneToMany field, so I thought thats correct way. The same way it is being used in other tests.
Anyhow I have updated and it works fine

Reply:
It's just `.search()` is returning list of results, no matter is there single result or multiple results :)

Reply:
@jyejare maybe you are confusing () with tuples creation. when you use () without any coma on it, it's just defining precedence:

```python
>>> (1) == 1
True
>>> type((1))
<class 'int'>
>>> type(1)
<class 'int'>
>>> 
>>> 1 + 6/2
4.0
>>> (1+6) /2
3.5
```
The issue is that parenthesis is also used for tuple creation. So how to create a tuple of just 1 element? You need to use a comma for this:

```python
>>> (1,) == 1
False
>>> type((1,))
<class 'tuple'>
>>> len((1,))
1
```
Of course this not the case on your tests.

Anyway sometimes you can see people using parenthesis to avoid the use of "\" when the line will have more than 79 chars. So maybe people started with code like this and forgot to remove the extra chars after refactoring it.